common/src/main/java/tachyon/security/PlainSaslServer.java

```diff
@@ -76,29 +75,36 @@ public String getMechanismName() {
         throw new IllegalArgumentException("Invalid message format, parts must contain 3 items");
       }
       String authzid = parts[0];
-      mAuthcid = parts[1];
+      String authcid = parts[1];
       String passwd = parts[2];
-      if (mAuthcid == null || mAuthcid.isEmpty()) {
+      if (authcid == null || authcid.isEmpty()) {
         throw new IllegalStateException("No authentication identity provided");
       }
       if (passwd == null || passwd.isEmpty()) {
         throw new IllegalStateException("No password provided");
       }
       if (authzid == null || authzid.isEmpty()) { // authzid = authcid
-        authzid = mAuthcid;
+        authzid = authcid;
```

Comment:
So currently we always assign authcid to authzid, right?
We should document this some where (maybe in the class description)?